src/server/api/schema.js

```diff
@@ -433,6 +437,32 @@ const rootMutations = {
 
       return updatedUser
     },
+    initiatePasswordReset: async (_, { organizationId, userId }, { user } ) => {
+      if (user.id !== userId) {
+        await accessRequired(user, organizationId, 'ADMIN', true)
+      }
+
+      const targetUser = await r
+        .table('user')
+        .get(userId)
+
+      const options = { method: 'POST',
+        url: `https://${process.env.AUTH0_DOMAIN}/dbconnections/change_password`,
+        headers: { 'content-type': 'application/json' },
+        body:
+          { client_id: `${process.env.AUTH0_CLIENT_ID}`,
+            email: targetUser.email,
+            connection: 'Username-Password-Authentication' },
+        json: true }
+
+      await request.post(options, (error, response, body) => {
+        const successful = !error && response.statusCode === 200
+        if (!successful) {
+          console.log(response.body)
```

Comment:
not sure what we should do here, but probably not just do response -- does request have a more async/await-friendly method?

Reply:
`request` does not return a promise. I thought about introducing a new library but in this instance it doesn't seem like a big deal. The user won't get the email is the worst case UX.

Reply:
Just curious here, not asking for a code change. Why `request` instead of `fetch`?